ocs_ci/deployment/deployment.py

```diff
@@ -3798,13 +3827,25 @@ def deploy(self):
             # Enable MCO console plugin
             enable_mco_console_plugin()
         config.switch_acm_ctx()
+        odf_running_version = version.get_ocs_version_from_csv(only_major_minor=True)
+        if odf_running_version >= version.VERSION_4_19:
+            # create service exporter
+            create_service_exporter()
+
         # RBD specific dr deployment
         if self.rbd:
             rbddops = RBDDRDeployOps()
             self.configure_mirror_peer()
             rbddops.deploy()
         self.enable_acm_observability()
+
+        if odf_running_version >= version.VERSION_4_19:
+            # validate storage cluster peer state
+            validate_storage_cluster_peer_state()
+
```

Comment:
storageclusterpeer will be create when drpolicy is created so this needs to be moved below self.deploy_dr_policy()

Reply:
done